modules/@apostrophecms/rich-text-widget/ui/apos/components/AposRichTextWidgetEditor.vue

```diff
@@ -536,6 +568,40 @@ export default {
     closeInsertMenuItem() {
       this.removeSlash();
       this.activeInsertMenuComponent = null;
+    },
+    closeInsertMenu(e) {
+      if (
+        [ 'ArrowUp', 'ArrowDown', 'Enter', ' ' ].includes(e.key) ||
+        this.activeInsertMenuComponent
+      ) {
+        return;
+      }
+      const pos = this.editor.view.state.selection.$anchor;
+      this.dismissedPositions.push({
+        pos: pos.pos,
+        parentOffset: pos.parentOffset
+      });
+      this.editor.commands.focus();
+      this.activeInsertMenuComponent = null;
+      // Only insert character keys
```

Comment:
This captures a character key press and appends it to the editor when returning focus, allowing uninterrupted typing when using a `/` character as real text. using `length === 1` to derive 'a normal character' might not be bulletproof ..